### DISCUSSION TITLE
How I can add new Label for the already exist Jira?

### DISCUSSION THREAD

**Discussion by dzaina** [andygrunwald/go-jira#629] [Q&A] [answered]
newLabel := map[string]interface{}{"update": map[string]interface{}{"labels": map[string]string{"add": "123"}}}
v, err := jiraClient.Issue.UpdateIssue(issue.Key, newLabel)

all time with error- take a look on body... can someone assist me?

**Comment by dzaina** [comment, marked as answer]
var lblarr []map[string]string
			lbl := map[string]string{"add": "NEW_LABEL"}
			lblarr = append(lblarr, lbl)
			body := map[string]interface{}{"update": map[string][]map[string]string{"labels": lblarr}}
			_, err := jiraClient.Issue.UpdateIssue(issue.Key, body)
			if err != nil {
				log.Println(err)
			}

**Comment by andygrunwald** [comment]
I re-open this as a reminder for me to add this to the documentation
